code/_helpers/icons.dm

```diff
@@ -1224,3 +1224,39 @@ lighting determines lighting capturing (optional), suppress_errors suppreses err
 
 	var/icon/I = getFlatIcon(thing)
 	return icon2html(I, target, sourceonly = sourceonly)
+
+/**
+ * Centers an image.
+ * Requires:
```

Comment:
Single newlines in DMdocs are not represented in the mouse over tooltip in VSC, you need an empty DMdoc line to go newline

Reply:
Ported 1:1 from /tg/, will fix